.github/workflows/exercism_test_helper_build_test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
 
     container:
-      image: elixir:1.13-slim
+      image: hexpm/elixir:1.14.0-erlang-25.0.4-debian-bullseye-20220801
```

Comment:
`elixir:1.14-slim` is a rc version, not the new official release, and stuff fails because 1.14 from mix.exs doesn't match 1.14-rc that's installed in the container, because that's considered to be a lower version.